commands/Dialog.js

```diff
@@ -0,0 +1,121 @@
+import Jimp from "jimp";
+import Discord from "discord.js";
+import {
+  EMOJI_REGEX,
+  EMOTE_MENTIONS_REGEX,
+  NONASCII_REGEX
+} from "./base/DiscordRegex";
+import Command from "./base/Command";
+
+import backgrounds from "../assets/images/dialog/bg/backgrounds.json";
+import characters from "../assets/images/dialog/char/characters.json";
```

Comment:
You could move these one folder up
```suggestion
import backgrounds from "../assets/images/dialog/backgrounds.json";
import characters from "../assets/images/dialog/characters.json";
```